### DISCUSSION TITLE
How can i add a custom function to the admin panel with a plugin, are there any hooks for the admin area ?

### DISCUSSION THREAD

**Discussion by andrelatino** [GetPublii/Publii#1756] [Help]
I would like to add a custom button on all pages with a plugin, help please

**Comment by bjazmoore** [comment]
What do you want the button to do?  You could use an anchor tag, give it a class and then style it as a button.

**Reply by andrelatino** [reply]
@bjazmoore Thanks for your reply! What I actually need is to create multiple plugins that will add custom options to the admin pages/posts. I’m not sure which admin functions exist to hook into, similar to how WordPress uses add_action('admin_head', 'myFunctionHere');. It would be great to have more documentation for developers like me who want to create plugins.

**Comment by bobmitro** [comment]
If you want to add a button to the front end, use the  post/page custom options: https://getpublii.com/dev/post-config-options/. To display custom options on the tag or author pages, you can refer to these links: https://getpublii.com/dev/tag-config-options/ and https://getpublii.com/dev/author-config-options/
